Makefile

```diff
@@ -888,7 +890,7 @@ buildbox-grpc: GOGOPROTO_IMPORTMAP := $\
 	Mignoreme=ignoreme
 buildbox-grpc:
 	@echo "PROTO_INCLUDE = $$PROTO_INCLUDE"
-	$(CLANG_FORMAT) -i -style='{ColumnLimit: 100, IndentWidth: 4, Language: Proto}' \
+	$(CLANG_FORMAT) -i -style=$(CLANG_FORMAT_STYLE) \
```

Comment:
Maybe add the options to the CLANG_FORMAT variable itself?

It's a bit messier than normal due to the way we "smartly" initialize variables, but something like this could do:

```make
ifeq ("$(CLANG_FORMAT)", "")
  CLANG_FORMAT := $(shell type -p clang-format clang-format-10 | head -1)
  CLANG_FORMAT := "$(CLANG_FORMAT) -i -style='{...}'"
endif
```

Reply:
I don't think wrapping it in `ifeq ("$(CLANG_FORMAT)", "")` would work, because the buildbox sets the `CLANG_FORMAT` env var.

https://github.com/gravitational/teleport/blob/1b1eaa40e34bc218a60e99eb3385f39c7f618924/build.assets/Makefile#L222-L234

I actually wanted to define `CLANG_FORMAT_STYLE` next to `CLANG_FORMAT`, but I soon realized I can't do that because the line that sets `CLANG_FORMAT` in the main Makefile is wrapped in a conditional.

As far as I understand, the `CLANG_FORMAT` var is used just to point at the clang-format binary, that's why I added a separate var with the style option.

Reply:
It looks like we don't need both the `which` fallback and the buildbox override.

That's all getting out of scope for your change, so I'll approve, but if you are feeling charitable feel free to update and give me a nudge for another pass.